ksql-api/src/main/java/io/confluent/ksql/api/server/KsqlInsertsException.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.server;
+
+import io.confluent.ksql.util.KsqlException;
+
+public class KsqlInsertsException extends KsqlException {
+
+  private final int errorCode;
```

Comment:
Would it make sense to have a base `KsqlApiException` that held the `errorCode`?  I'm sure there will be other operations other than inserts that want to return a error code, right?

Reply:
ack